src/testing/code-debugging.md

```diff
@@ -886,41 +886,45 @@ You can set these flags either by editing the framework code,
 or by importing the module and setting the value in your
 `main()` method, following by a hot restart.
 
-<dl markdown="1">
-<dt markdown="1">[`debugDumpRenderTree()`][]</dt>
-<dd markdown="1">
-  Call this function when not in a layout or repaint
+[`debugDumpRenderTree()`][]</dt>
+: Call this function when not in a layout or repaint
   phase to dump the rendering tree to the console.
   (Pressing **t** from `flutter run` calls this command.)
   Search for "RepaintBoundary" to see diagnostics
   on how useful a boundary is.
+
 {% comment %} Feature is not yet added to DevTools:
     Rather than using this flag to dump the render tree
     to a file, view the render tree in the Flutter inspector.
     To do so, bring up the Flutter inspector and select the
-    **Render Tree** tab. {% endcomment %}
-<dt markdown="1">[`debugPaintLayerBordersEnabled`][]</dt>
-<dd markdown="1">PENDING
-<dt markdown="1">[`debugRepaintRainbowEnabled`][]</dt>
-<dd markdown="1">You can enable this flag in the Flutter
-    inspector by selecting the **Highlight Repaints** button.
-    If any static widgets are rotating through the colors of the rainbow
-    (for example, a static header), those areas are candidates for adding
-    repaint boundaries.
-<dt markdown="1">[`debugPrintMarkNeedsLayoutStacks`][]</dt>
-<dd markdown="1">Enable this flag if you're seeing more layouts
-    than you expect (for example, on the timeline, on a profile,
-    or from a `print` statement inside a layout method).
-    Once enabled, the console is flooded with stack traces
-    showing why each render object is being marked dirty for
-    layout. You can use the `debugPrintStack()` method from the
-    `services` library to print your own stack traces on demand,
-    if this kind of approach is useful to you.
-<dt markdown="1">[`debugPrintMarkNeedsPaintStacks`][]</dt>
-<dd markdown="1">Similar to `debugPrintMarkNeedsLayoutStacks`,
-    but for excess painting. You can use the `debugPrintStack()`
-    method from the `services` library to print your own stack
-    traces on demand, if this kind of approach is useful to you.
+    **Render Tree** tab.
+{% endcomment %}
+
+[`debugPaintLayerBordersEnabled`][]</dt>
+: PENDING
```

Comment:
If you have a chance, can you make an issue for documenting this? Otherwise it might stay as pending indefinitely.